sagemaker-pipelines/step-decorator/computer-vision-examples/computer-vision-pipeline.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        "./dataset-full/",
</pre><p> </p><p>Nit: we can use the variable <code>local_dataset_folder</code> here in case the folder name is changed in the future</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Done